app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java

```diff
@@ -93,8 +96,9 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
     private void initViews() {
         rankingListAdapter = new RankingListAdapter(this);
         rankingRecyclerView.setAdapter(rankingListAdapter);
-        rankingRecyclerView.setNestedScrollingEnabled(false);
+        rankingRecyclerView.setNestedScrollingEnabled(false); 
```

Comment:
After extensive reviewing and discussion, I've came along an extra character placed at the end of the semicolons in line 99. Although this has not a direct impact on code functionality, it affects the base coding guidelines we are following.